values.html

```diff
@@ -0,0 +1,188 @@
+						communities across the world. We also believe that through these
+						values is where we can find an authentic future with digital
+						technology.
+					</p>
+				</div>
+			</div>
+		</section>
+		<!-- Values-list section -->
+		<div class="desktop-parent">
+			<!-- Desktop Group 1 START -->
+			<div class="desktop-group">
+				<section class="copperback values-card">
+					<div class="text-container">
+						<h2 class="h2-styles">Humanity and Inclusion</h2>
+						<p>
+							We believe in human rights, equity, inclusion, and fostering
+							belonging for all people to include native and indigenous
+							communities.
+						</p>
+					</div>
+				</section>
+				<section class="sustainability values-card onyback">
+					<img
+						src="./assets/images/pages/values/mobile/sustainability-m.jpg"
+						alt="a person holding a tablet in the middle of nature"
+					/>
+					<div class="sustainability-text text-container">
+						<h2 class="h2-styles">Sustainability</h2>
+						<p>
+							We support long term sustainability and responsible treatment of
+							our natural environment and its resources.
+						</p>
+					</div>
+				</section>
+				<!-- Desktop Group 1 END -->
+			</div>
+			<div class="desktop-group">
+				<section class="authenticity values-card">
+					<div class="authenticity-text text-container">
+						<h2 class="h2-styles">Authenticity</h2>
+						<p>
+							We believe in authentic representation and utility for indigenous
+							communities.
+						</p>
+					</div>
+					<img
+						src="./assets/images/pages/values/mobile/authenticity-m.jpg"
+						alt="an indigenous family laughing together"
+					/>
+				</section>
+
+				<section class="grapeback values-card text-container">
+					<h2 class="h2-styles">Multigenerational Connection</h2>
+					<p>
+						We support multigenerational needs and beliefs that are important to
+						indigenous or native communities.
+					</p>
+				</section>
+			</div>
+			<!-- Desktop Group END -->
+			<div class="desktop-group">
+				<section class="education values-card">
+					<div class="education-text text-container">
+						<h2 class="h2-styles">Education</h2>
+						<p>
+							We support and encourage STEAM education where it is needed and
+							appropriate for native and indigenous communities.
+						</p>
+					</div>
+					<img
+						src="./assets/images/pages/values/mobile/education-m.jpg"
+						alt="an aboriginal woman working"
+					/>
+				</section>
+				<section class="technology values-card">
+					<div class="tech-text text-container">
+						<h2 class="h2-styles">Technology and Innovation</h2>
+						<p>
+							We advocate responsible stewardship with digital tech &
+							innovation. Technologies should not be leveraged to harm native
+							and indigenous communities or lands.
+						</p>
+					</div>
+				</section>
+			</div>
+		</div>
+		<!-- Support section -->
+		<section class="support">
```

Comment:
This sections looks like it can use the global **.wrapper** to ensure that content inside the support section follows those gutters we mentioned earlier in class!

Please add in a wrapper inside the section to ensure that the content has a width and max width!

![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/2d8efc1a-7f12-4d19-844c-f546043bc0a3)

for reference, take a look at some resources: 
[Slides](https://docs.google.com/presentation/u/0/d/1YuvT9kffo2NBi_dZQHZ7rAWCsBFiC2Htto3nQr66E0A/edit)
[CSS Tricks Guide ](https://css-tricks.com/best-way-implement-wrapper-css/)

Reply:
Done, I thnk? Line 139

 `.support {
	background-color: var(--color-yaleblue);
	--wrapper-width-max: 1328px;
	--desktop-wrapper-width: 90%;
	--mobile-wrapper-width: 95%;
}`

Reply:
The wrapper width looks good on desktop, but the spacing looks off - not enough spacing on top and the content should be vertically centered, according to the design. Please double-check the font sizes as well, the heading looks small. 

<img width="926" alt="Screen Shot 2024-01-24 at 8 01 22 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/a687b88a-ebfb-4d32-888f-903a40685373">


Also, it looks like the section needs more padding on mobile. 

<img width="250" alt="Screen Shot 2024-01-24 at 8 30 10 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/c7788cd2-fc9c-40a5-8c90-bf99dd98ea7c">